libs/translations/src/data/supported-translations.ts

```diff
@@ -0,0 +1,1567 @@
+  },
+  {
+    name: 'Swedish',
+    locale: 'sv-SE',
+    crowdinCode: 'sv-SE',
+  },
+  {
+    name: 'Swedish, Finland',
+    locale: 'sv-FI',
+    crowdinCode: 'sv-FI',
+  },
+  {
+    name: 'Syriac',
+    locale: 'syc-SY',
+    crowdinCode: 'syc',
+  },
+  {
+    name: 'Tagalog',
+    locale: 'tl-PH',
+    crowdinCode: 'tl',
+  },
+  {
+    name: 'Tahitian',
+    locale: 'ty-PF',
+    crowdinCode: 'ty',
+  },
+  {
+    name: 'Tajik',
+    locale: 'tg-TJ',
+    crowdinCode: 'tg',
+  },
+  {
+    name: 'Talossan',
+    locale: 'tzl-TZL',
+    crowdinCode: 'tzl',
+  },
+  {
+    name: 'Tamil',
+    locale: 'ta-IN',
+    crowdinCode: 'ta',
+  },
+  {
+    name: 'Tatar',
+    locale: 'tt-RU',
+    crowdinCode: 'tt-RU',
+  },
+  {
+    name: 'Telugu',
+    locale: 'te-IN',
+    crowdinCode: 'te',
+  },
+  {
+    name: 'Tem (Kotokoli)',
+    locale: 'kdh-KDH',
+    crowdinCode: 'kdh',
+  },
+  {
+    name: 'Thai',
+    locale: 'th-TH',
+    crowdinCode: 'th',
+  },
+  {
+    name: 'Tibetan',
+    locale: 'bo-BT',
+    crowdinCode: 'bo-BT',
+  },
+  {
+    name: 'Tigrinya',
+    locale: 'ti-ER',
+    crowdinCode: 'ti',
+  },
+  {
+    name: 'Tsonga',
+    locale: 'ts-ZA',
+    crowdinCode: 'ts',
+  },
+  {
+    name: 'Tswana',
+    locale: 'tn-ZA',
+    crowdinCode: 'tn',
+  },
+  {
+    name: 'Turkish',
+    locale: 'tr-TR',
+    crowdinCode: 'tr',
+  },
+  {
+    name: 'Turkish, Cyprus',
+    locale: 'tr-CY',
+    crowdinCode: 'tr-CY',
+  },
+  {
+    name: 'Turkmen',
+    locale: 'tk-TM',
+    crowdinCode: 'tk',
+  },
+  {
+    name: 'Ukrainian',
+    locale: 'uk-UA',
+    crowdinCode: 'uk',
+  },
+  {
+    name: 'Upper Sorbian',
+    locale: 'hsb-DE',
+    crowdinCode: 'hsb-DE',
+  },
+  {
+    name: 'Urdu (India)',
+    locale: 'ur-IN',
+    crowdinCode: 'ur-IN',
+  },
+  {
+    name: 'Urdu (Pakistan)',
+    locale: 'ur-PK',
+    crowdinCode: 'ur-PK',
+  },
+  {
+    name: 'Uyghur',
+    locale: 'ug-CN',
+    crowdinCode: 'ug',
+  },
+  {
+    name: 'Uzbek',
+    locale: 'uz-UZ',
+    crowdinCode: 'uz',
+  },
+  {
+    name: 'Valencian',
+    locale: 'val-ES',
+    crowdinCode: 'val-ES',
+  },
+  {
+    name: 'Venda',
+    locale: 've-ZA',
+    crowdinCode: 've',
+  },
+  {
+    name: 'Venetian',
+    locale: 'vec-IT',
+    crowdinCode: 'vec',
+  },
+  {
+    name: 'Vietnamese',
+    locale: 'vi-VN',
+    crowdinCode: 'vi',
+  },
+  {
+    name: 'Walloon',
+    locale: 'wa-BE',
+    crowdinCode: 'wa',
+  },
+  {
+    name: 'Welsh',
+    locale: 'cy-GB',
+    crowdinCode: 'cy',
+  },
+  {
+    name: 'Wolof',
+    locale: 'wo-SN',
+    crowdinCode: 'wo',
+  },
+  {
+    name: 'Xhosa',
+    locale: 'xh-ZA',
+    crowdinCode: 'xh',
+  },
+  {
+    name: 'Yiddish',
+    locale: 'yi-DE',
+    crowdinCode: 'yi',
+  },
+  {
+    name: 'Yoruba',
+    locale: 'yo-NG',
+    crowdinCode: 'yo',
+  },
+  {
+    name: 'Zeelandic',
+    locale: 'zea-ZEA',
+    crowdinCode: 'zea',
+  },
+  {
+    name: 'Zulu',
+    locale: 'zu-ZA',
+    crowdinCode: 'zu',
```

Comment:
I would stay independent from crowdin in the naming.

It's quite classic to have 2 letters codes.